docs/reference/controls/itemscontrol.md

```diff
@@ -17,9 +18,34 @@ To see the full list of _Avalonia UI_ built-in repeating data controls, see [her
 
 You will probably use these properties most often:
 
-<table><thead><tr><th width="316">Property</th><th>Description</th></tr></thead><tbody><tr><td><code>ItemsSource</code></td><td>The bound collection that is used as the data source for the control.</td></tr><tr><td><code>ItemsControl.ItemTemplate</code></td><td>Attached property element to contain the data template for an individual item. </td></tr></tbody></table>
-
-## Example
+<table>
+  <thead>
+    <tr>
+      <th width="316">Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>ItemsSource</code></td>
+      <td>The bound collection that is used as the data source for the control.</td>
+    </tr>
+    <tr>
+      <td><code>ItemsControl.ItemTemplate</code></td>
+      <td>Attached property element to contain the data template for an individual item.</td>
+    </tr>
+    <tr>
+      <td><code>ItemsControl.ItemPanel</code></td>
+      <td>Parent panel to display the items in. By default, this is a StackPanel</td>
+    </tr>
+    <tr>
+      <td><code>ItemsControl.Styles</code></td>
+      <td>The style that is applied to each individual item.</td>
```

Comment:
This is not technically correct. It can be used to apply to an item but really it will work with any child element of the items control